build.gradle

```diff
@@ -1,13 +1,51 @@
+ext {
+    coroutinesVersion = '1.5.2'
+    wordPressUtilsVersion = '2.2.0'
+    wordPressLoginVersion = '0.0.8'
+    gutenbergMobileVersion = 'v1.68.0'
+    storiesVersion = '1.2.0'
+    aboutAutomatticVersion = '0.0.2'
+
+    minSdkVersion = 24
+    compileSdkVersion = 31
+    targetSdkVersion = 30
+
+    coroutinesVersion = '1.3.9'
+    androidxWorkVersion = "2.4.0"
+
+    daggerVersion = '2.29.1'
+    fluxCVersion = '1.32.0'
+
+    appCompatVersion = '1.0.2'
+    coreVersion = '1.3.2'
+    lifecycleVersion = '2.2.0'
+    constraintLayoutVersion = '1.1.3'
+    materialVersion = '1.2.1'
+    preferenceVersion = '1.1.0'
+    swipeToRefresh = '1.1.0'
+    uCropVersion = '2.2.4'
+    lifecycleVersion = '2.2.0'
```

Comment:
Not related to the changes, but looks like there is a duplicate of `lifecycleVersion`

Reply:
Yeap looks like it, good catch 👍 I am working in a different branch right now, would you like to suggest removing one of them from Github UI so I can approve it?